src/js/components/DataTable/Resizer.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { findDOMNode } from 'react-dom';
+
+import { Box } from '../Box';
+
+export default class Resizer extends Component {
+  state = {};
+
+  ref = React.createRef();
+
+  onMouseDown = (event) => {
+    if (this.ref.current) {
+      const element = findDOMNode(this.ref.current);
+      const rect = element.getBoundingClientRect();
+      this.setState({ start: event.clientX, width: rect.width }, () => {
+        document.addEventListener('mousemove', this.onMouseMove);
+        document.addEventListener('mouseup', this.onMouseUp);
+      });
+    }
+  }
+
+  onMouseMove = (event) => {
+    const { property } = this.props;
+    const { width } = this.state;
+    const nextWidth = Math.max(12, width + (event.clientX - this.state.start));
```

Comment:
why `12` ? can we at least add a comment?

Reply:
Will do.